src/Mixins/Expectation.php

```diff
@@ -502,6 +502,18 @@ public function toBeNumeric(string $message = ''): self
         return $this;
     }
 
+    /**
+     * Asserts that the value contains only digits.
+     *
+     * @return self<TValue>
+     */
+    public function toBeDigits(string $message = ''): self
```

Comment:
Does Pest or Vitest have an equivalent on this?

Reply:
> Does Pest or Vitest have an equivalent on this?

According to my review, none of them have an equivalent for this natively, including as Pest's `toBeNumeric` is using [PHP is_numeric](https://www.php.net/manual/en/function.is-numeric.php) the test below will result in success:

```php
expect('9001 ')->toBeNumeric(); //space
expect('1.23e5')->toBeNumeric(); //dot
expect('-35')->toBeNumeric(); //negative
```

**Manually**, the alternative to this would be:

```php
expect(ctype_digit('9001 '))->toBeTrue(); //false
expect(ctype_digit('1.23e5'))->toBeTrue(); //false
expect(ctype_digit('-35'))->toBeTrue(); //false
```

With addition of `toBeDigits` we have ensure that the string contains only digits.